meson.build

```diff
@@ -305,9 +299,10 @@ configure_file(input : 'include/config.h.in',
 # We expect these libs to supplied with the distribution
 curl_dep = dependency('libcurl', version: '>=7.19.0')
 
-if osdet == 'darwin' and not get_option('nix')
-  libcrypto = meson.get_compiler('c').find_library('crypto', dirs: [ '/usr/local/opt/openssl/lib/' ])
-  libssl = meson.get_compiler('c').find_library('ssl', dirs: [ '/usr/local/opt/openssl/lib/' ])
+if (osdet == 'darwin' and not get_option('nix') and
+    run_command('test', '-d', '/usr/local/opt/openssl/lib').returncode() == 0)
```

Comment:
@benjamin-tlon, does this remove the need for the `nix` option? Or should stay so it can work even in the presence of a homebrew installation?